etc/config/c++.amazon.properties

```diff
@@ -561,3 +561,11 @@ libs.catch2.versions.222.version=2.2.2
 libs.catch2.versions.222.path=/opt/compiler-explorer/libs/catch2/v2.2.2/include
 libs.catch2.versions.223.version=2.2.3
 libs.catch2.versions.223.path=/opt/compiler-explorer/libs/catch2/v2.2.3/include
+libs.doctest.name=Doctest
+libs.doctest.description=The fastest feature-rich C++11 single-header testing framework for unit tests and TDD 
+libs.doctest.url=https://github.com/onqtam/doctest
+libs.doctest.versions=trunk:129
+libs.doctest.versions.trunk.version=trunk
+libs.doctest.versions.trunk.path=/opt/compiler-explorer/libs/doctest/trunk/doctest
+libs.doctest.versions.129.version=1.2.9
+libs.doctest.versions.129.path=/opt/compiler-explorer/libs/doctest/1.2.9/doctest
```

Comment:
Can you add back a newline at eof please?

Reply:
Wups. No problem :)